src/main/java/org/example/gameService/VCSFileCreator.java

```diff
@@ -0,0 +1,53 @@
+package org.example.gameService;
+
+import org.example.model.GameData;
+
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.Map;
+
+public class VCSFileCreator {
+
+    private LocalTime time;
+    private final DateTimeFormatter formatter;
+
+    public VCSFileCreator() {
+
+        this.time = LocalTime.now();
+        this.formatter = DateTimeFormatter.ofPattern("HH:mm:ss");
+    }
+
+    private Map<String, Integer> VCSFormatSchema() {
+
+        Map<String, Integer> schema = new HashMap<>();
+        schema.put("houseScore", 10);
+        schema.put("playerScore", 11);
+        schema.put("playerPoints", 12);
+        schema.put("changeInPoints", 14);
+        schema.put("playerWin", 9);
+        schema.put("sesja z godziny", 26);
+        return schema;
+    }
+
+    public String createVCSContent(GameData gameData) {
```

Comment:
could this method be static?

Reply:
Previously, I quckly created the VCSFileCreator class, which resulted in poor considered fields and poor structured methods in both the SaveGame and VCSFileCreator classes. In the current commit, I have cleaned up the code by removing unnecessary fields and improving the methods.